src/models/userModel.ts

```diff
@@ -0,0 +1,26 @@
+import { Column, Entity, PrimaryGeneratedColumn } from "typeorm";
+@Entity()
+class User {
+  @PrimaryGeneratedColumn()
+  public id: string;
+
+  @Column()
+  public firstName: string;
+
+  @Column()
+  public lastName: string;
+
+  @Column()
+  public country: string;
```

Comment:
`country`,`city`, not required, we need `Role`

Reply:
on it!

Reply:
On that note, how many roles should we have? I propose two, an `is_admin` column whose value is a boolean